.github/std-unexploded.yml

```diff
@@ -106,3 +106,48 @@ jobs:
       - uses: divnix/std-action/run@main
         with: {ffBuildInstructions: true, remoteStore: "ssh-ng://eu.nixbuild.net"}
 
+  deploy-to-eu:
+    <<: *job
+    needs: [discover, images]
+    name: ${{ matrix.target.jobName }} (eu-central-1)
+    env:
+      AWS_REGION: eu-central-1
+    permissions:
+      id-token: write
+      contents: read
+      deployments: write
+    environment:
+      name: dev-preview
+      url: https://backend.dev-preview.eks.lw.iog.io
+    if: fromJSON(needs.discover.outputs.hits).deployments.apply != '{}'
+    strategy:
+      matrix:
+        target: ${{ fromJSON(needs.discover.outputs.hits).deployments.apply }}
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@main
```

Comment:
Once we stabilize this repo Ill expect for all these inputs to be pinned

Reply:
That's an oversight, `v2.0.0` had been broken for a short while back then.

Reply:
Addressed in [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/10167f20f9c7d8e8870f84755ab2e0bd1a902e2f..47ce6e3ec1f74f224a253a6c05706323d0dd3ba9)